servconf.c

```diff
@@ -661,8 +661,8 @@ derelativise_path(const char *path)
 		return xstrdup("none");
 	expanded = tilde_expand_filename(path, getuid());
 #ifdef WINDOWS
-        /* Windows absolute paths - \abc, /abc, c:\abc, c:/abc*/
-	if (*expanded == '/' || *expanded == '\\' ||
+        /* Windows absolute paths - \abc, /abc, c:\abc, c:/abc, __PROGRAMDATA__\openssh\sshd_config */
+	if (*expanded == '/' || *expanded == '\\' || strstr(expanded, SSHDIR) ||
```

Comment:
hmm.. this is a nasty side effect and will not look good for upstream integration.

Reply:
Moved this logic to misc.c.. defined in stdio.h